mteb/tasks/Classification/heb/Hebrew_Sentiment_Analysis.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification  # type: ignore
+from mteb.abstasks.TaskMetadata import TaskMetadata  # type: ignore
+
+
+class Hebrew_Sentiment_Analysis(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="Hebrew_Sentiment_Analysis",
+        dataset={
+            "path": "hebrew_sentiment",
```

Comment:
You'll also need to specify the subset name with `name="..."`, there are two subsets `morph` and `token`.
What's the difference between them and which one should be used?

Reply:
There isn't too much difference between the 2 subsets. Essentially, certain words are changed in the tweet/text (only one word is changed in the entire tweet, and it is changed with a synonymous word. ex : President -> Leader).
I have used the morph subset